### PR TITLE
Update the README #4154

### LINKED ISSUE #4154: Update the README
## Description of your problem
The readme (landing page) has some out-of-date information.

Most importantly, https://discourse.pymc.io is not linked prominently enough.

The code of conduct should probably be linked too.

### PR DESCRIPTION
**Pull request by Overlord2335**
Updated README file 
- Added PyMC discourse link in top paragragh.
- Added Code of conduct link in Contributors section.

**Thank your for opening a PR!**

Depending on what your PR does, here are a few things you might want to address in the description:
+ [ ] what are the (breaking) changes that this PR makes?
+ [ ] important background, or details about the implementation
+ [ ] are the changes—especially new features—covered by tests and docstrings?
+ [ ] consider adding/updating relevant example notebooks
+ [ ] right before it's ready to merge, mention the PR in the RELEASE-NOTES.md

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -13,6 +13,7 @@ large suite of problems.
 Check out the `getting started guide <http://docs.pymc.io/notebooks/getting_started>`__,  or
 `interact with live examples <https://mybinder.org/v2/gh/pymc-devs/pymc3/master?filepath=%2Fdocs%2Fsource%2Fnotebooks>`__
 using Binder!
+Or you can also check out `PyMC Discourse <https://discourse.pymc.io/>`__ which place to discuss Bayesian Computation using PyMC3 (current version) and probabilistic modelling in general.
```
Comment (twiecki) [inline README.rst]:
```suggestion
For questions on PyMC3, head on over to our `PyMC Discourse <https://discourse.pymc.io/>`__ forum.
```

**Review by twiecki — APPROVED** [review]

**Comment by twiecki** [comment]
Thanks @Overlord2335!

**Comment by Overlord2335** [comment]
@twiecki When will the branch be merged? Or do I have to make some further changes before that?

**Comment by Overlord2335** [comment]
@twiecki  Do I have to change anything else too as the issue is not yet closed?

**Comment by twiecki** [comment]
Can you link the issue?

**Comment by Overlord2335** [comment]
[Here](https://github.com/pymc-devs/pymc3/issues/4154) is the issue, @twiecki

**Comment by Overlord2335** [comment]
@twiecki Could you point me to a open issue that I could work on as a beginner and some material to read regarding the issue too, Thanks.

**Comment by MarcoGorelli** [comment]
> [Here](https://github.com/pymc-devs/pymc3/issues/4154) is the issue, @twiecki

Just for the future, if you put
```
closes #<issue number>
```
in the pull request description, then that issue will be closed automatically when the PR is merged

**Pull request merged**